package.json

```diff
@@ -1,37 +1,51 @@
 {
 	"name": "obsidian-tts",
-	"version": "0.5.3",
+	"version": "0.5.5",
 	"description": "Text to speech for Obsidian. Hear your notes.",
 	"main": "main.js",
 	"scripts": {
-		"dev": "node esbuild.config.mjs",
-		"build": "node esbuild.config.mjs production",
+		"dev": "rollup --config rollup.config.dev.mjs -w",
+		"build": "rollup --config rollup.config.build.mjs",
```

Comment:
Please keep esbuild.

Reply:
For this to work, we can't use esbuild. 
esbuild does not work for bundling AudioWorklet. The worklets are used for soundtouch which is used to control the playback rate and pitch for OpenAI since they do not provide a native way of controlling these parameters at the time of speech generation like SpeechSynthesis and Azure do so the worklets and rollup are needed to circumvent that limitation.

Reply:
Any thoughts on this?

Reply:
wow confirmed the guy just ignore people.

Reply:
It's easy to ban people when in fact you are the problem...